fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1279,6 +1279,44 @@ Future< Standalone<RangeResultRef> > ReadYourWritesTransaction::getRange(
 		}
 	}
 	
+	// Use special key prefix "\xff\xff/transaction/conflicting_keys/<some_key>",
+	// to retrieve keys which caused latest not_committed(conflicting with another transaction) error.
+	// The returned key value pairs are interpretted as :
+	// prefix/<key1> : '1' - any keys equal or larger than this key are (probably) conflicting keys
+	// prefix/<key2> : '0' - any keys equal or larger than this key are (definitely) not conflicting keys
+	// Currently, the conflicting keyranges returned are original read_conflict_ranges or union of them.
+	// TODO : This interface needs to be integrated into the framework that handles special keys' calls in the future
+	if (begin.getKey().startsWith(conflictingKeysAbsolutePrefix) && end.getKey().startsWith(conflictingKeysAbsolutePrefix)) {
+		// Remove the special key prefix "\xff\xff"
+		KeyRef beginConflictingKey = begin.getKey().removePrefix(specialKeys.begin);
+		KeyRef endConflictingKey = end.getKey().removePrefix(specialKeys.begin);
+
+		// Check if the conflicting key range to be read is valid
+		KeyRef maxKey = getMaxReadKey();
+		if(beginConflictingKey > maxKey || endConflictingKey > maxKey)
+			return key_outside_legal_range();
+		
```

Comment:
This is an empty line with spaces/tabs. Can you clang-format the PR before we merge it?

Reply:
Sure. Thanks.